### PR TITLE
Project1

### PR DESCRIPTION
**Pull request by sergiogval**
This PR add the skeleton of the app, it includes the style as the template does, also it includes some features regarding project2. Specifically the one that add a new task.

### CODE REVIEW

File: src/print.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <button class="positioner" type="button">
+      <span>&#8729;</span>
+      <span>&#8729;</span>
+      <span>&#8729;</span>
+      </button>
+      </li>`
+      input.value = ''
+  }
+ 
+}
```
Comment (Emmyn5600) [inline src/print.js:32]: - [ ] In this milestone, we are supposed to implement this. Kindly, implement the following as listed. 

- You need an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number].

```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```
- [ ]  It has a function that iterates over the tasks array and populates the HTML list item element for each task.

- [ ] On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @sergiogval 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️
- Check the comments under the review.
- Try to add linters even if some are failing I know it can be fixed.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/print.js (hunk 1 above)
Comment (sergiogval) [reply src/print.js]: Ok, I solved the linters and make a default task appear in the DOMloaded event. I will appreciate your feedback.

**Review by eri8-9 — APPROVED** [review]
Hi @sergiogval , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**